tests/test_auxtel_focus_sweep_latiss.py

```diff
@@ -81,6 +81,51 @@ async def test_configure(self):
             assert self.script.config.grating == 1
             assert self.script.config.n_images_per_step == 1
 
+    async def test_configure_focus_steps(self):
+        config = {
+            "axis": "x",
+            "focus_steps": [0, 100, 200, 300, 400],
```

Comment:
This is kind of a picky comment, but the scale here is pretty far off from what we normally do with the real hexapods. Focus steps are provided to the axes in units of mm, so we usually do steps of 0.1mm or less. Would you mind updating these tests to something like [0, 0.1, 0.2, 0.3, 0.4]?

Reply:
Great comment! I was unsure of what values to use here so I looked at the rest of the tests that were already written and tried to use similar ones. I've changed all tests written by me to use these values and changed simple ``assert``s to ``assertAlmostEqual`` to deal with floating point representation. Let me know if you want me to update older tests too.

Reply:
Actually I checked with Tiago on this one and the simonyi hexapod offsets are given in microns, not mm like they are in AuxTel. So hundreds of microns actually makes sense for some of these unit tests (the comcam and lsstcam versions). Sorry, I was only thinking of auxtel when I made this comment.